apix-rest-v1/src/main/java/eu/xenit/apix/rest/v1/internal/ApixArgumentResolver.java

```diff
@@ -1,6 +1,7 @@
 package eu.xenit.apix.rest.v1.internal;
```

Comment:
So I assume this whole class can be removed after https://xenitsupport.jira.com/browse/ALFREDAPI-476 ?

Reply:
that would be the goal of that ticket, yes, Though it's up to you and Thijs whether we actually go though with that.
@kerkhofsd I consider removing the resolver out of scope for this ticket. Unless you've got a strong argument for removing it now, just going with the modification seems most efficient to me.

Reply:
I agree that removing the `ApixArgumentResolver` in favor of the `@RequestBody` functionality is a different scope.

Reply:
Or let me correct: defining the scope of this ticket not up to me. 
I agree that removing the `ApixArgumentResolver` in favor of the `@RequestBody` is a considerable amount of work that can be handled separately.

Reply:
Yes, out of scope

Reply:
Not sure if you tagged me here because of my initial remark. => See additional comment there.